linux_os/guide/services/deprecated/package_telnetd_removed/rule.yml

```diff
@@ -16,15 +16,14 @@ identifiers:
     cce@rhel7: CCE-82461-5
 
 references:
-    anssi: NT007(R03)
     nist: CM-7(a),CM-7(b),CM-6(a)
     isa-62443-2013: 'SR 1.1,SR 1.10,SR 1.11,SR 1.12,SR 1.13,SR 1.2,SR 1.3,SR 1.4,SR 1.5,SR 1.6,SR 1.7,SR 1.8,SR 1.9,SR 2.1,SR 2.2,SR 2.3,SR 2.4,SR 2.5,SR 2.6,SR 2.7,SR 3.1,SR 3.5,SR 3.8,SR 4.1,SR 4.3,SR 5.1,SR 5.2,SR 5.3,SR 7.1,SR 7.6'
     isa-62443-2009: 4.3.3.5.1,4.3.3.5.2,4.3.3.5.3,4.3.3.5.4,4.3.3.5.5,4.3.3.5.6,4.3.3.5.7,4.3.3.5.8,4.3.3.6.1,4.3.3.6.2,4.3.3.6.3,4.3.3.6.4,4.3.3.6.5,4.3.3.6.6,4.3.3.6.7,4.3.3.6.8,4.3.3.6.9,4.3.3.7.1,4.3.3.7.2,4.3.3.7.3,4.3.3.7.4,4.3.4.3.2,4.3.4.3.3
     cobit5: APO13.01,BAI10.01,BAI10.02,BAI10.03,BAI10.05,DSS01.04,DSS05.02,DSS05.03,DSS05.05,DSS06.06
     iso27001-2013: A.11.2.6,A.12.1.2,A.12.5.1,A.12.6.2,A.13.1.1,A.13.2.1,A.14.1.3,A.14.2.2,A.14.2.3,A.14.2.4,A.6.2.1,A.6.2.2,A.9.1.2
     cis-csc: 11,12,14,15,3,8,9
     nist-csf: PR.AC-3,PR.IP-1,PR.PT-3,PR.PT-4
-    anssi: BP28(R1)
+    anssi: BP28(R1),NT007(R03)
```

Comment:
So in some cases you remove NT references, but in this case (and some other cases too), you have decided to keep it. What is the pattern?

Reply:
There are more ANSSI guides relevant than just BP28. NT007 in this case is for SSH.

Reply:
OK, thank you for the explanation!